10LessonTen/Checkpoint2/index.html

```diff
@@ -6,5 +6,31 @@
 </head>
 <body>
 
+  <div class="container">
+    <header>
+      <nav>
+        <span class="sabl">San Antonio's Best Listings</span>
+        <ul>
+          <li>Contact</li>
+          <li>About Us</li>
+          <li>Our Partners</li>
+        </ul>
+      </nav>
+    </header>
+    <section id="leftBox">
+      <img src="http://data.whicdn.com/images/44249774/superthumb.jpg"/>
+    </section>
+
+    <section id="rightBox">
+      <article>
+        <h3>Alamo Realty</h3>
+        <p>Etiam pharetra, erat sed fermentum frugiat, velit mauris egestas quam, ut aliquam massa nisl quis neque. Suspendisse in orci enim.</p>
+      </article>
+      <article>
+        <p> <a href="service.html"><button type="button" style="width:100px; height:50px;"name="CONTACT US ">CONTACT US ></button></a> </p>
+      </article>
+    </section>
+
+  </div>
 </body>
 </html>
```

Comment:
It looks like you still need to finish up with the checkpoint 2 assignment. I will grade it when you are finished.